appengine/cloudsql/app.yaml

```diff
@@ -21,14 +21,16 @@ env_variables:
   MYSQL_USER: [YOUR_USER]
   MYSQL_PASSWORD: [YOUR_PASSWORD]
   MYSQL_DATABASE: [YOUR_DATABASE]
-  # This path was printed to the console when you started the proxy, and is of
-  # the format: `/[DIR]/[YOUR_PROJECT_ID]:[YOUR_REGION]:[YOUR_INSTANCE_NAME]`
-  MYSQL_SOCKET_PATH: [YOUR_SOCKET_PATH]
+  # The instance connection name can be determined from this command:
+  # `gcloud sql instances describe [INSTANCE_NAME]` or by viewing the
+  # Instance details page in the Google Cloud Platform Console.
```

Comment:
I think the concatenation approach helps understand what to look for. It could be confused with project id at first glance.